config/initializers/menus.rb

```diff
@@ -42,7 +42,7 @@
             last: true
   menu.push :help, OpenProject::Info.help_url,
             last: true,
-            caption: I18n.t('label_help'),
+            caption: '',
```

Comment:
Yeah, that one's not pretty. I can extend the Redmine menu system to accept a separate title attribute.

Currently, caption is used both as the content and the title of the menu item.